### PR TITLE
Add Southampton Feburary Tech Day Event

### PR DESCRIPTION
**Pull request by WillB97**

### CODE REVIEW

File: _events/sr2023/soton-tech-day-february.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+|-------|------|
+| 10:00 | Doors Open. |
+| 12:30 | There will be a lunch break. Teams should bring their own lunches or get them from food vendors near the venues. |
+| 13:00 | Lightning Talks, where teams will talk about their robots and what they plan on doing with them. |
+| 17:00 | Finish. |
+
+[tech-day-signup]: https://forms.gle/oJ9DK8imzek7fQQb6
+[soton-food-map]: https://goo.gl/yYlfs5
+[soton-directions]: {{ site.baseurl }}/resources/sr2020/southampton-tech-day-directions-highfield-building-25.pdf
+[soton-campus-directions]: http://www.southampton.ac.uk/about/visit/getting-to-our-campuses.page
```
Comment (PeterJCLaw) [inline _events/sr2023/soton-tech-day-february.md:44]: Can we make this link TLS? (I had thought that this was checked in CI, though it seems not).

Comment (PeterJCLaw) [reply _events/sr2023/soton-tech-day-february.md:44]: > Can we make this link TLS? (I had thought that this was checked in CI, though it seems not).

```suggestion
[soton-campus-directions]: https://www.southampton.ac.uk/about/visit/getting-to-our-campuses.page
```

Comment (PeterJCLaw) [reply _events/sr2023/soton-tech-day-february.md]: f53f19ccedb7ccb8111538607e2be7075710d447

File: _events/sr2023/southampton-tech-day-february.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+layout: event
+type: tech-day
+location: Southampton University
+---
+
+Tech Days are opportunities for teams to spend a whole day working on their
+robot with lots of help available. They're also an opportunity to see how other
+teams are doing or get more direct help with your robots.
+
+We provide a space for you to work in, with power and internet access, as well
```
Comment (WillB97) [inline _events/sr2023/southampton-tech-day-february.md:14]: @Adimote what is the situation with internet access?

File: _events/sr2023/southampton-tech-day-february.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+as volunteers able to help you with your kits and hands-on guidance with your
+robots.
+
+Please [let us know][tech-day-signup] by Friday 27th January 2023 if you'd like to attend.
+
+At the request of the venue:
+
+* any power tools must be PAT tested,
```
Comment (WillB97) [inline _events/sr2023/southampton-tech-day-february.md:22]: Can we clarify that they need to be PAT tested before arrival.

**Review by WillB97 — COMMENTED** [review]
Looks good, minor clarifications

**Comment by PeterJCLaw** [comment]
Closing for now as this isn't happening. If we go ahead in march we can probably recover this then.

**Pull request closed without merging**